cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -259,6 +259,57 @@ bool SubstraitToVeloxPlanValidator::validateLiteral(
   return true;
 }
 
+bool SubstraitToVeloxPlanValidator::isDeniedCast(
```

Comment:
Preferred function name: `isSuppportedCast`. Then return false for unsupported case.